docroot/modules/humsci/hs_admin/hs_admin.module

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/**
+ * @file
+ * Contains hs_admin.module.
+ */
+
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Implements hook_form_alter().
+ */
+function hs_admin_form_alter(&$form, FormStateInterface $form_state, $form_id) {
```

Comment:
I realize this was the initial technical direction I added to the ticket but after review it in action and doing some research I think we can simplify further. 
```
<?php

/**
 * @file
 * Contains hs_admin.module.
 */

/**
 * Implements hook_form_alter().
 */
function hs_admin_preprocess_node_edit_form(&$variables) {
  $variables['#attached']['library'][] = 'hs_admin/hs_admin';
}
```
The above only preprocesses the form to attach the library instead of using the form_alter function. This task really isn't altering the form itself so the preprocess function is more appropriate. Also, by targeting the `node_edit_form` we affect all content types.